### DISCUSSION TITLE
Icons in TOC items?

### DISCUSSION THREAD

**Discussion by markpiller** [squidfunk/mkdocs-material#7248] [Q&A] [answered]
Hello, can someone point me to the docs describing how to configure icons for the TOC items as shown https://squidfunk.github.io/mkdocs-material/reference/ ?

Thanks in advance!

**Comment by alexvoss** [comment, marked as answer]
These are [page icons](https://squidfunk.github.io/mkdocs-material/reference#setting-the-page-icon).

**Reply by markpiller** [reply]
Thank you!
